hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/search/search.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Input, Select } from 'antd';
+import { Option } from '@/v2/components/select/singleSelect';
+
+type SearchProps = {
+  searchColumn?: string;
+  searchOptions?: Option[];
+  onSearch: (
+    arg0: string,
+    arg1: React.ChangeEvent<HTMLInputElement>
+    | React.MouseEvent<HTMLElement, MouseEvent>
+    | React.KeyboardEvent<HTMLInputElement>
+    | undefined
+  ) => void;
+  onChange: (
+    value: string,
+    //OptionType, OptionGroupData and OptionData are not
+    //currently exported by AntD hence set to any
+    option: any
+  ) => void;
+}
+
+const Search: React.FC<SearchProps> = ({
+  searchColumn,
+  searchOptions,
+  onSearch = () => {},
+  onChange = () => {}
+}) => {
+
+  let selectFilter = null;
+
+  if (searchColumn) {
+    selectFilter = <Select
+      defaultValue={searchColumn}
+      options={searchOptions}
+      onChange={onChange} />
+  }
```

Comment:
```suggestion
  const selectFilter = searchColumn ? (
  <Select
    defaultValue={searchColumn}
    options={searchOptions}
    onChange={onChange}
  />
) : null;

```

Reply:
Addressed in the latest commit [1347224](https://github.com/apache/ozone/pull/7048/commits/13472245263b08e28aeda28b1f3597df5abc3396)